lib/ansible/modules/network/panos/panos_restart.py

```diff
@@ -104,10 +104,9 @@ def main():
     except Exception:
         x = sys.exc_info()[1]
         if 'succeeded' in str(x):
-            module.exit_json(changed=True, msg=str(msg))
```

Comment:
I get that.  And I can make these changes.  However my goal here is to do very targeted changes to resolve the undefined errors, not necessarily do py3 compat.